xmodule/library_content_block.py

```diff
@@ -587,15 +587,20 @@ def children_are_syncing(self, request, suffix=''):  # pylint: disable=unused-ar
             is_updating = False
         return Response(json.dumps(is_updating))
 
-    def studio_post_duplicate(self, store, source_block):
+    def studio_post_duplicate(
+        self,
+        source_item,
```

Comment:
Why did you replace `source_block` with `source_item`? A "block" is a much more verbose term than a generic "item".

Reply:
because that was the variable name of the code this function was refactored from. i left it as item because i wasn't sure that the `source_item` being passed was a block based on the original code.

Reply:
renamed!